app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/OperationsMenuController.kt

```diff
@@ -496,35 +496,35 @@ class OperationsMenuController(args: Bundle) : BaseController(
                     )
                 }
             }
-            binding.resultImageView.visibility = View.VISIBLE
+            binding?.resultImageView?.visibility = View.VISIBLE
```

Comment:
In places like this it makes more sense to do `binding?.let{...}` and then you have a non-null reference inside, instead of repeating the null checks. But I understand this PR is just about getting it to work without NPEs first.